### PR TITLE
Add support for Elastic Stack 7.5.0

### LINKED ISSUE #323: Add support Elastic Stack 7.5.0

### PR DESCRIPTION
**Pull request by russcam**
This PR adds support for Elastic Stack 7.5.0.

As part of this PR, the VM SKUs within the ARM template have been updated from now deprecated VM SKUs to newer SKU offerings. The documentation has been updated in line with these changes.

Elastic Stack 5.x has now been removed from the template in this version, allowing for simplification of the deployment scripts.

Closes #323

### CODE REVIEW

**Comment by russcam** [comment]
Links need to be updated before merging

```sh
npm run links -- --branch master
```

File: README.md [hunk 1]
```diff
@@ -670,16 +670,16 @@ the `_artifactsLocation` parameter of the template to point to a specific tagged
 
 **Targeting a specific template version is recommended for repeatable production deployments.**
 
-For example, to target the [`7.0.0` tag release with PowerShell](https://github.com/elastic/azure-marketplace/tree/7.0.0)
+For example, to target the [`7.4.0` tag release with PowerShell](https://github.com/elastic/azure-marketplace/tree/7.4.0)
```
Comment (codebrain) [inline README.md]: 7.5.0?

File: README.md [hunk 2]
```diff
@@ -670,16 +670,16 @@ the `_artifactsLocation` parameter of the template to point to a specific tagged
...
 
 ```powershell
-$templateVersion = "7.0.0"
+$templateVersion = "7.4.0"
```
Comment (codebrain) [inline README.md]: 7.5.0?

File: README.md [hunk 3]
```diff
@@ -670,16 +670,16 @@ the `_artifactsLocation` parameter of the template to point to a specific tagged
...
 $_artifactsLocation = "https://raw.githubusercontent.com/elastic/azure-marketplace/$templateVersion/src/"
 
 # minimum parameters required to deploy
 $clusterParameters = @{
   "_artifactsLocation" = $_artifactsLocation
-  "esVersion" = "7.0.0"
+  "esVersion" = "7.4.0"
```
Comment (codebrain) [inline README.md]: 7.5.0?

File: docs/faqs.asciidoc [hunk 4]
```diff
@@ -33,13 +32,13 @@ The cost of deploying from the Marketplace will vary depending on the cluster
...
 you decide to deploy to. Because there are a number of variables involved, a cost
-needs to be calculated on a case-by-case basis. At a minimum, you will pay for
+needs to be calculated on a case-by-case basis. **At a minimum**, you will pay for
 all Azure resources deployed by the template. The
 {azurecalculator}[Azure pricing calculator] can help in providing a good estimate
 for this. In addition to the cost of Azure resources,
 {subscriptions}[Elastic Stack Gold, Platinum and Enterprise features] require a
 license to be purchased, the cost of which is dependent on a number of factors.
-Please get in touch with us to discuss.
+{contact}[Please get in touch with us to discuss].
```
Comment (codebrain) [inline docs/faqs.asciidoc]: Call to action; website link? e-mail address?

Comment (russcam) [reply docs/faqs.asciidoc]: It's an attribute defined in  `docs/azure-marketplace.asciidoc`

File: README.md (hunk 3 above)
Comment (russcam) [reply README.md]: Doesn't need to be as it's just an example, but have updated

File: README.md (hunk 2 above)
Comment (russcam) [reply README.md]: Doesn't need to be as it's just an example, but have updated

File: README.md (hunk 1 above)
Comment (russcam) [reply README.md]: Doesn't need to be as it's just an example, but have updated

Comment (russcam) [reply README.md]: Doesn't need to be as it's just an example, but have updated

File: README.md (hunk 3 above)
Comment (russcam) [reply README.md]: Doesn't need to be as it's just an example, but have updated

File: README.md (hunk 1 above)
Comment (codebrain) [reply README.md]: ++ only version 7.4.2 exists, I would just bump to latest always.

File: build/allowedValues.json [hunk 5]
```diff
@@ -1,17 +1,17 @@
 {
   "versions": [
-    "5.6.16",
```
Comment (Mpdreamz) [inline build/allowedValues.json]: :+1:

5.6.x | 2019-03-11
-- | --

File: src/empty/empty-jumpbox-resources.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```
Comment (Mpdreamz) [inline src/empty/empty-jumpbox-resources.json]: Just out of curiousity what prompted the bump?

File: src/scripts/elasticsearch-install.sh [hunk 7]
```diff
@@ -290,7 +290,7 @@ else
     UNICAST_HOSTS="${UNICAST_HOSTS%?}]"
 fi
 
-if [[ $(dpkg --compare-versions "$ES_VERSION" "ge" "6.0.0"; echo $?) -eq 0 && (${INSTALL_XPACK} -ne 0 || ${BASIC_SECURITY} -ne 0) ]]; then
```
Comment (Mpdreamz) [inline src/scripts/elasticsearch-install.sh]: 💥

File: src/scripts/elasticsearch-install.sh [hunk 8]
```diff
@@ -430,17 +425,6 @@ plugin_cmd()
     echo /usr/share/elasticsearch/bin/elasticsearch-plugin
 }
 
-install_xpack()
```
Comment (Mpdreamz) [inline src/scripts/elasticsearch-install.sh]: 💥 👋

File: src/empty/empty-jumpbox-resources.json (hunk 6 above)
Comment (russcam) [reply src/empty/empty-jumpbox-resources.json]: The visual studio code ARM template extension prompted me 🙂

**Comment by russcam** [comment]
Rebased and squashed some commits so that they can be rebased and merged

**Pull request merged**